src/node/stage/approved.rs

```diff
@@ -1001,20 +1002,23 @@ impl Approved {
             // malicious elders within one side don't carry out vote, triggering the sibling
             // section's DKG process by send them an Initial DKG message.
             if !dkg_key.0.contains(core.id()) {
-                trace!(
-                    "Triggering DKG process for sibling section during splitting {:?}",
-                    dkg_key.0
-                );
+                // Avoiding the case that a staled elder trigger DKG votes by mistake.
+                if num_of_new_infos > 1 {
```

Comment:
Not sure about this. Sometimes it can happen that the new elders are completely different from the old ones even outside of split (for example, due to relocation) and we still need to notify them.

Reply:
Also, could this be the reason for the failure that happens when we relocate all elders?

Reply:
>Sometimes it can happen that the new elders are completely different from the old ones even outside of split (for example, due to relocation) and we still need to notify them.
>could this be the reason for the failure that happens when we relocate all elders?

Yes, I think so. and even worse, the NotifyOldElder message won't be picked up, as at that time, the node is still using the old-name, not the new-name. Meanwhile the `EldersInfo` is already looking for the new names. :(